### DISCUSSION TITLE
What are the best programs to make slides in computing? / ¿Cuáles son los mejores programas para hacer diapositivas en informática?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#121531] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

# 👋 Hola
Mi pregunta es si conocen algún buen programa para mostrar código.

# 👋 Hey 
My question is if you know of any good programs to display code.

**Comment by KrlitosForever** [comment, marked as answer]
Hola considero que los mejores programas para aquello son:

- [Quarto](https://github.com/quarto-dev)
- [Revealjs](https://github.com/reveal/revealjs.com)
- [Sli.dev](https://github.com/slidevjs/slidev),

Espero haberte ayudado

**Reply by ghost** [reply]
Muchas Gracias.
